rollup.config.js

```diff
@@ -0,0 +1,55 @@
+import nodeResolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import uglify from 'rollup-plugin-uglify'
+import {sizeSnapshot} from 'rollup-plugin-size-snapshot'
+
+const input = './src/index.js'
+
+const name = 'jss'
```

Comment:
we can read it from the package.json automatically, right?

Reply:
It's usually not the same name. package.json contains dashed name. This `name` is camel cased for window object. So not sure we should reuse it.

Reply:
we can camelize

Reply:
It's not always fit to camalize. For example there is cytoscape.js project which creates Cytoscape global variable, or react-powerplug which creates ReactPowerPlug global variable. It's better to set it manually.

Reply:
In our case it will work, because naming is consistent.